packages/drivers/routerlicious-driver/src/wholeSummaryDocumentStorageService.ts

```diff
@@ -233,4 +253,24 @@ export class WholeSummaryDocumentStorageService implements IDocumentStorageServi
         });
         await Promise.all(blobCachePutPs);
     }
+
+    private convertSnapshotAndBlobsToSummaryTree(
```

Comment:
nit: This does not need to be part of the class.

Reply:
Moving this to tree utils for now. We have a separate convo where tree utils should be.